package.json

```diff
@@ -15,71 +15,65 @@
     "watch:build": "tsc -p tsconfig.json -w",
-    "watch:test": "nyc --silent ava --watch",
-    "cov": "run-s build test:unit cov:html cov:lcov && open-cli coverage/index.html",
-    "cov:html": "nyc report --reporter=html",
-    "cov:lcov": "nyc report --reporter=lcov",
-    "cov:send": "run-s cov:lcov && codecov",
-    "cov:check": "nyc report && nyc check-coverage --lines 100 --functions 100 --branches 100",
+    "watch:test": "NODE_NO_WARNINGS=1 jest --watch",
     "doc": "run-s doc:html && open-cli build/docs/index.html",
     "doc:html": "typedoc --out build/docs",
     "doc:json": "typedoc --json build/docs/typedoc.json",
     "doc:publish": "gh-pages -m \"[ci skip] Updates\" -d build/docs",
     "version": "standard-version",
     "reset-hard": "git clean -dfx && git reset --hard && npm i",
     "prepare-release": "run-s reset-hard test cov:check doc:html version doc:publish",
-    "generate-types": "typechain --target ethers-v5 --out-dir 'types/ethers-v5' './node_modules/@lukso/universalprofile-smart-contracts/build/contracts/*.json'",
-    "postinstall": "npm run generate-types"
+    "generate-types": "typechain --target ethers-v5 --out-dir 'types/ethers-v5' './node_modules/@lukso/universalprofile-smart-contracts/build/contracts/*.json'"
   },
   "engines": {
     "node": ">=10"
   },
   "dependencies": {
+    "@erc725/erc725.js": "^0.6.2-beta.4",
     "@ethersproject/abi": "^5.4.0",
-    "@nomiclabs/hardhat-ethers": "^2.0.2",
-    "erc725.js": "^0.5.7",
-    "ethers": "5.4.1",
-    "graphql": "^15.5.1",
-    "typedoc-neo-theme": "^1.1.1"
+    "browser-image-compression": "^1.0.15",
+    "ethers": "5.4.5",
+    "ipfs-http-client": "52.0.2",
+    "ipfs-utils": "^8.1.6"
   },
   "devDependencies": {
-    "@ava/typescript": "^1.1.1",
-    "@istanbuljs/nyc-config-typescript": "^1.0.1",
+    "@ethereum-waffle/jest": "^3.3.0",
+    "@lukso/universalprofile-smart-contracts": "0.1.1",
+    "@nomiclabs/hardhat-ethers": "^2.0.2",
     "@typechain/ethers-v5": "7.0.1",
-    "@types/chai": "^4.2.21",
-    "@types/mocha": "^8.2.3",
+    "@types/jest": "^27.0.1",
     "@types/node": "^16.4.0",
     "@typescript-eslint/eslint-plugin": "^4.0.1",
     "@typescript-eslint/parser": "^4.0.1",
-    "ava": "^3.12.1",
-    "chai": "^4.3.4",
     "codecov": "^3.5.0",
     "cspell": "^4.1.0",
     "cz-conventional-changelog": "^3.3.0",
+    "esbuild": "^0.12.24",
+    "esbuild-jest": "^0.5.0",
     "eslint": "^7.31.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.23.4",
-    "gh-pages": "^3.1.0",
-    "hardhat": "^2.4.3",
+    "gh-pages": "^3.2.3",
```

Comment:
Maybe you want to add this link https://lukso-network.github.io/tools-lsp-factory/ in the README ?

Is it manual publish? I don't see an action for this.

Reply:
We'll publish on the docs-website, but that's going to happen a little bit later.